pkgs/development/cuda-modules/generic-builders/manifest.nix

```diff
@@ -46,7 +46,7 @@ let
   # redistArch :: String
   # The redistArch is the name of the architecture for which the redistributable is built.
   # It is `"unsupported"` if the redistributable is not supported on the target platform.
-  redistArch = flags.getRedistArch hostPlatform.system;
+  redistArch = flags.getRedistArch targetPlatform.system;
```

Comment:
Why? I think we generally want the host system's binaries, except when we handle `lib/` and `nvvm/lib/` in `nvcc` (then we do want target)